### ISSUE TITLE
framework

### ISSUE THREAD

**Issue by albanie** [DavyNeven/fastSceneUnderstanding#5]
Nice work. Out of interest, are there any plans for a release in any other frameworks?

**Comment by dbbert** [comment]
Not at the moment, but we know of someone who is working on a Tensorflow implementation. We'll post a link when it's there!

**Comment by albanie** [comment]
Great, thanks!

**Comment by uyekt** [comment]
@dbbert  Do you know if this someone has already implemented your discriminative loss function for instance segmentation in Tensorflow? 
I'd like to test your loss on my network for a project ending soon but I'm not sure if I'll have enough time to rewrite it myself, since I'm neither fluent in Lua nor Torch.

**Comment by JadTawil-theonly** [comment]
any updates on this?
